docs/index.rst

```diff
@@ -47,25 +47,20 @@ Using funcX
 ^^^^^^^^^^^^^^^^^^
 
 funcX offers a Python SDK for registering, sharing, and executing functions.
-The following code shows how funcX can be used to register and run a "hello world"
-function on a remote endpoint.
-
+The following code block examples how funcX can be used to execute a "hello
+world" function on a remote endpoint.
 
 .. code-block:: python
 
-    from funcx.sdk.client import FuncXClient
-
-    fxc = FuncXClient()
+    from funcx.sdk.client import FuncXExecutor
```

Comment:
Can we change to `from funcx import FuncXExecutor`?

Reply:
Good call; absolutely.